rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -61,12 +61,12 @@
 encryption_key = hashlib.md5(password).hexdigest()
 
 
-def test_exec(config, ssh_con):
+def test_exec(config):
```

Comment:
Any reason why ssh_con was removed

I think it was added as a part of PR: Adds support script execution from non-RGW node

Reply:
yeah, not removed intentionally, will update the branch.